lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -361,16 +361,8 @@ public function getListTableIndexesSQL($table, $currentDatabase = null)
      */
     private function getTableWhereClause($table, $classAlias = 'c', $namespaceAlias = 'n')
     {
-        $whereClause = $namespaceAlias.".nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast') AND ";
-        if (strpos($table, ".") !== false) {
```

Comment:
IIRC we require this logic for full qualified table names. You can't simply leave this out.